website/content/en/docs/reference/configuration/unit-tests.md

```diff
@@ -284,6 +284,56 @@ test conditions using a special configuration-based system. `check_fields` is no
 strongly recommend converting any existing `check_fields` tests to `vrl` conditions.
 {{< /danger >}}
 
+#### Asserting no output
+
+In some cases, you may need to assert that _no_ event is output by a transform. You can specify
+this at the root level of a specific test's configuration using the `no_outputs_from` parameter,
+which takes a list of transform names. Here's an example:
+
+```toml
+[[tests]]
+name = "Ensure no output"
+no_outputs_from = ["log_filter", "metric_filter"]
+```
+
+In this test configuration, Vector would expect that the `log_filter` and `metric_filter` transforms
+dont't output _any_ events.
+
+Some examples of use cases for `no_outputs_from`:
+
+* When testing a [`filter`][filter] transform, you may want to assert that the [input](#inputs)
+  event is filtered out
+* When testing a [`remap`][remap] transform, you may need to assert that VRL's [`abort`][abort]
```

Comment:
This item isn't rendering correctly:

![image](https://user-images.githubusercontent.com/11903873/141794058-4f866722-7d7c-4b47-ab79-2464765bbef0.png)

Reply:
Looks like that function isn't documented. I'll fix that in a follow-up PR.